CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic
+Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
+
+
+## [Unreleased]
+
+### Added
+
+### Changed
+
+### Fixed
+
+## [1.0.0] - 2024-02-09
+
+The first release with a number of fixes. Since `libp2p/openssl` is not
```

Comment:
Let's add a link to `libp2p/openssl` releases in case one decide to check CHANGELOG from the beginning at least to commit message/PR description.

https://github.com/libp2p/go-openssl/releases

Reply:
Thank you for the idea, added.